rust/kcl-lib/src/std/chamfer.rs

```diff
@@ -46,44 +51,57 @@ async fn inner_chamfer(
             source_ranges: vec![args.source_range],
         }));
     }
-
+    if tags.is_empty() {
+        return Err(KclError::Semantic(KclErrorDetails {
+            source_ranges: vec![args.source_range],
+            message: "You must chamfer at least one tag".to_owned(),
+        }));
+    }
     let mut solid = solid.clone();
-    for edge_tag in tags {
-        let edge_id = match edge_tag {
-            EdgeReference::Uuid(uuid) => uuid,
-            EdgeReference::Tag(edge_tag) => args.get_tag_engine_info(exec_state, &edge_tag)?.id,
-        };
-
-        let id = exec_state.next_uuid();
-        args.batch_end_cmd(
-            id,
-            ModelingCmd::from(mcmd::Solid3dFilletEdge {
-                edge_id,
-                object_id: solid.id,
-                radius: LengthUnit(length.to_mm()),
-                tolerance: LengthUnit(DEFAULT_TOLERANCE), // We can let the user set this in the future.
-                cut_type: CutType::Chamfer,
-            }),
-        )
-        .await?;
+    let edge_ids: Vec<_> = tags
+        .into_iter()
+        .map(|edge_tag| edge_tag.get_engine_id(exec_state, &args))
+        .collect::<Result<Vec<_>, _>>()?;
 
+    let id = exec_state.next_uuid();
+    let mut extra_face_ids = Vec::new();
+    let num_extra_ids = edge_ids.len() - 1;
+    for _ in 0..num_extra_ids {
+        extra_face_ids.push(exec_state.next_uuid());
+    }
+    let strategy = strategy.unwrap_or_default();
```

Comment:
We should warn or error on a bad strategy, at the moment a typo in a string literal will silently use the default